### ISSUE TITLE
Wrong status text in ImageView from DebugVisualizer

### ISSUE THREAD

**Issue by fdncred** [accord-net/framework#897]
**What would you like to submit?** (put an 'x' inside the bracket that applies)

  - [ ] question
  - [x] bug report
  - [ ] feature request

**Issue description**
I found a bug that I was trying to fix in the DebugVisualizer but I cannot open the Solution or CSProj in VS2017 in order to fix it.  So, I can't compile my fix.

For what it's worth, I think this is the fix in ImageView.cs like 232. If you zoom in, the X,Y position is wrong in the status strip. *Untested* code below.
```cs
    private void pbPreview_MouseMove(object sender, MouseEventArgs e)
    {
        if (e.Button == MouseButtons.Left && (pMain.HorizontalScroll.Visible || pMain.VerticalScroll.Visible))
        {
            System.Drawing.Point scrollOffset = new System.Drawing.Point(e.X - dragStartPosition.X, e.Y - dragStartPosition.Y);
            pMain.AutoScrollPosition = new System.Drawing.Point(-pMain.AutoScrollPosition.X - scrollOffset.X, -pMain.AutoScrollPosition.Y - scrollOffset.Y);
        }

        tsslX.Text = $"X: {e.X / Zoom}";
        tsslY.Text = $"Y: {e.Y / Zoom}";
    }
```

**Comment by cesarsouza** [comment]
Hi Darren,

Thanks a lot for the fix! Please, let me know which errors do you get when trying to open the solution or DebugVisualizers as well so I can look into it!

Regards,
Cesar

**Comment by fdncred** [comment]
If I try to open the .sln file, nothing opens. I suspect this is because the .sln refers to a file that no longer exists "DebuggerVisualizers.csproj".  So then when I try to open the Accord.Debug.Visualizers (GPL).csproj I get this error.


Microsoft Visual Studio
The imported project "D:\Some\Path\Accord.Net\Tools\DebuggerVisualizers\Accord.NET.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.  D:\Some\Path\Accord.Net\Tools\DebuggerVisualizers\Accord.Debug.Visualizers (GPL).csproj
OK

**Comment by cesarsouza** [comment]
I see - this is because I had edited ```DebuggerVisualizers.csproj``` to use the same configuration parameters  (output path, build events, etc) as the rest of the framework. Those are specified in the Accord.NET.targets file that is actually contained in the framework's solution folder, but not in this project's folder in particular.

One solution is to open the main solution for the framework, ```Accord.NET.sln```, located in the Sources folder. Or, I can try to put a workaround so it becomes possible to open at least this project separately from the others if needed. I might look into this.

Regards,
Cesar

**Comment by fdncred** [comment]
Oh, I didn't think the DebugVisualizer was in the main solution. It opens there. I'll try to fiddle around with it to see if my fix worked.

**Comment by fdncred** [comment]
@cesarsouza It looks to me like this fixes the problem. I kind of hate to go to the hassle of making a PR when there's such a minor change.

**Comment by cesarsouza** [comment]
No worries I can put it in in one of my next commits. Thanks!

**Comment by cesarsouza** [comment]
Fixed in 3.8.0.
